stories/About/Developer Guide.mdx

```diff
@@ -32,3 +32,24 @@ import client from '../assets/client_component.png';
   <code>git push --set-upstream origin richard/user-picks-page</code>
 
   Subsequent pushes will now go to your remote upstream branch.
+
+## Connecting to <del>Supabase</del>
```

Comment:
I think we need to talk about merge conflicts. :D This is not the first time Supabase came back into your PR.